go/vt/vtgateproxy/vtgateproxy.go

```diff
@@ -174,4 +179,8 @@ func (proxy *VTGateProxy) StreamExecute(ctx context.Context, session *vtgateconn
 }
 
 func Init() {
+	log.V(100).Infof("Registering GRPC dial options")
```

Comment:
eek - I think demmer and I talked about this and never fixed it - thank you for moving this here and doing it once!